foundation/foundation-upgrade/src/test/resources/specifications/v1_8_0/orphedomos-to-fabric8-migration.feature

```diff
@@ -24,3 +24,55 @@ Feature: Migrate from using Orphedomos to Fabric8 docker-maven-plugin
     When the Fabric8 migration executes
     And the POM is not modified
 
+  Scenario: A POM with the Docker packaging type includes the docker-maven-plugin in the build
+    Given a POM with the "docker-build" packaging type
+    And the docker-maven-plugin is not already in the build section
+    When the add Fabric8 migration executes
+    Then the docker-maven-plugin is added to the build section with minimal configuration
+
+  Scenario: Add plugin, no plugins section
+    Given a POM with the "docker-build" packaging type
+    And there is no plugins section in build
+    When the add Fabric8 migration executes
+    Then the docker-maven-plugin is added to the build section with minimal configuration
+
+  Scenario: Add plugin, no build section
+    Given a POM with the "docker-build" packaging type
+    And there is no build section
+    When the add Fabric8 migration executes
+    Then the docker-maven-plugin is added to a new build section with minimal configuration
+
+  Scenario: Add plugin, no build section
```

Comment:
A: maybe a different scenario name: skip migration if already in build section?

Reply:
updated all the titles to be more clear